styles.css

```diff
@@ -0,0 +1,84 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
```

Comment:
- [ ] According to the project requirements, your webpage should have the background like the given Figma design:

![image](https://user-images.githubusercontent.com/92215878/204805819-49660218-0fbc-4d26-b3cf-a8431999b558.png)